http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AzurePubsubHandler.scala

```diff
@@ -168,6 +176,104 @@ class AzurePubsubHandlerInterp[F[_]: Parallel](
       _ <- wsmDao.createVm(createVmRequest, auth)
     } yield ()
 
+  override def startAndMonitorRuntime(runtime: Runtime, azureCloudContext: AzureCloudContext)(implicit
+    ev: Ask[F, AppContext]
+  ): F[Unit] = for {
+    ctx <- ev.ask
+    monoOpt <- azureVmServiceInterp.startAzureVm(InstanceName(runtime.runtimeName.asString), azureCloudContext)
+
+    _ <- monoOpt match {
+      case None =>
+        logger.error(ctx.loggingCtx)(
+          s"Cannot find runtime ${runtime.projectNameString}."
+        ) // Error? For now, log a message
+        F.raiseError(
+          RuntimeNotFoundException(
```

Comment:
we should raise `AzureRuntimeStartingError` so that it'll be handled properly in the handler. And you can get rid of the log statement above